packages/lex-cli/src/codegen/lex-gen.ts

```diff
@@ -127,11 +129,17 @@ export function genObject(
           if (propDef.items.type === 'ref') {
             propAst = iface.addProperty({
               name: `${propKey}${req ? '' : '?'}`,
-              type: `${refToType(
-                propDef.items.ref,
-                stripScheme(stripHash(lexUri)),
-                imports,
-              )}[]`,
+              type: makeType(
+                refToType(
+                  propDef.items.ref,
+                  stripScheme(stripHash(lexUri)),
+                  imports,
+                ),
+                {
+                  nullable: propNullable,
+                  array: true,
```

Comment:
just out of curiosity, what does this `array: true` do?

Reply:
An example of that would be the difference between `A | B` and `(A | B)[]`.